daemon/volumes_windows.go

```diff
@@ -25,7 +25,7 @@ func (daemon *Daemon) setupMounts(c *container.Container) ([]container.Mount, er
 		if err := daemon.lazyInitializeVolume(c.ID, mount); err != nil {
 			return nil, err
 		}
-		s, err := mount.Setup(c.MountLabel, idtools.IDPair{0, 0}, nil)
+		s, err := mount.Setup(c.MountLabel, idtools.Identity{UID: 0, GID: 0, SID: ""}, nil)
```

Comment:
In general, I'm not sure it's worthwhile to initialize fields (`UID`, `GID`, and `SID`, here) to default values. The code was doing that before this change, but it seems unnecessary since Go already initializes to those values.